04-advanced-topics/network-analysis_rnaseq_01_wgcna.Rmd

```diff
@@ -399,14 +412,14 @@ WGCNA's authors recommend using a `power` that has an signed $R^2$ above `0.80`,
 If you have multiple power values with signed $R^2$ above `0.80`, then picking the one at an inflection point, in other words where the $R^2$ values seem to have reached their saturation [@Zhang2005].
 You want to a `power` that gives you a big enough $R^2$ but is not excessively large.
 
-So using the plot above, going with a power soft-threshold of `16`!
+So using the plot above, going with a power soft-threshold of `2`!
```

Comment:
Okey dokey. So `7`? Also how do we explain the rationale for `7`?

Reply:
It is an inflection point (max). If you look at their example, they pick the first max, not just the first point above 0.8 which seems to align well here.

Reply:
It seems like such a subtle inflection, I did not notice.

Reply:
They main point is that we can go a fair bit above the level we were at with 2, so it is worth setting higher. I doubt it would be much different at 5 or 6...